test/ussd_registration.test.js

```diff
@@ -275,15 +280,15 @@ describe("Mama Nigeria App", function() {
                         state: 'state_baby_birth_month_year',
                         reply: [
                             "Select the month & year the baby was born:",
-                            "1. July 15",
-                            "2. June 15",
-                            "3. May 15",
-                            "4. Apr 15",
-                            "5. Mar 15",
-                            "6. Feb 15",
-                            "7. Jan 15",
-                            "8. Dec 14",
-                            "9. Nov 14"
+                            "1. April 15",
+                            "2. March 15",
+                            "3. February 15",
+                            "4. January 15",
+                            "5. December 14",
+                            "6. November 14",
+                            "7. October 14",
+                            "8. September 14",
+                            "9. More"
```

Comment:
Please add tests for the expected values when `9. More` is selected for both of the month display options.